manifests/overlays/apps/apps-addon/nvidia/gpu-quickstart.yaml

```
@@ -12,12 +12,12 @@ spec:
   description: Install GPU add-on
   prerequisites: [Your cluster has at least one GPU node provisioned.]
   introduction: |-
-    This quick start shows you how to install the GPU add-on and verify that the JupyterHub spawner detects the available GPUs.
+    This quick start shows you how to install the GPU add-on and verify that the Jupyter spawner detects the available GPUs.
```

```suggestion
    This quick start shows you how to install the GPU add-on and verify that the Jupyter Spawner detects the available GPUs.
```

Initcap for Spawner (not spawner)?